apps/crn-server/test/app.test.ts

```diff
@@ -142,15 +145,13 @@ describe('Contentful feature flag', () => {
   });
 
   describe('Dashboard Data Provider', () => {
-    const dashboardSquidexDataProviderMock: jest.Mocked<DashboardDataProvider> =
-      {
-        fetch: jest.fn(),
-      };
-
-    const dashboardContentfulDataProviderMock: jest.Mocked<DashboardDataProvider> =
-      {
-        fetch: jest.fn(),
-      };
+    const dashboardSquidexDataProviderMock = {
+      fetch: jest.fn(),
+    } as unknown as jest.Mocked<DashboardSquidexDataProvider>;
```

Comment:
why are you having to cast?

Reply:
I can't really remember, but as the dashboard provider does not implement the generic data provider i will revert the change.